core/pack.go

```diff
@@ -14,6 +14,9 @@ import (
 // Pack stores the modpack metadata, usually in pack.toml
 type Pack struct {
 	Name  string `toml:"name"`
+	Author string `toml:"author"`
+	Version string `toml:"version"`
+	ProjectID int `toml:"project_id"`
```

Comment:
This could probably moved to the curseforge module somehow, maybe add a field to the pack:
```go
Export   map[string]toml.Primitive `toml:"export"`
```
then the curseforge module can parse that into a project ID, similar to how updates work.

In the TOML it could look like this:
```
[export.curseforge]
project-id = 1000
```